### ISSUE TITLE
Support for Unity editor

### ISSUE THREAD

**Issue by andreparodi-bandai** [rudderlabs/rudder-sdk-unity#18]
Hi,
Is there any support for sending events when running inside the editor?
Thanks
Andre

**Comment by desusai7** [comment]
Hey @andreparodi-bandai, 

By saying editor, do you mean to say while running the unity app on the desktop, if that's the case you cannot send events form desktop as the unity SDK supports only Android and iOS platforms at the moment

**Comment by jmarbach** [comment]
> By saying editor, do you mean to say while running the unity app on the desktop, if that's the case you cannot send events form desktop as the unity SDK supports only Android and iOS platforms at the moment

Yes, that's correct. Please add support for the Unity desktop editor. This will enable developers will be able to troubleshoot and validate events more quickly in development.

In order to emit new events to RudderStack today, the RudderStack Unity SDK requires a new build to be loaded in Android or iOS, which may take considerable time depending on the size of the game. By adding support for Unity desktop editor, wait time for feedback on changes to RudderStack components will be eliminated.
